_posts/2018-02-16-NewPipe-Beta-now-available.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: NewPip Beta now available
+short: NewPipe Beta published
+date: 2018-02-16 16:00:00 +0100
+categories: [announcement, pinned]
+author: TobiGr
+image: newpipe-beta
+excerpt_separator: <!-- more -->
+---
+
+With the aim of making NewPipe bug free and achieving a stable 1.0.0 release,
+we are pleased to announce the immediate availability of NewPipe Beta.
+
+<!-- more -->
+
+The beta app can be downloaded and installed via [F-Droid](https://f-droid.org/packages/org.schabi.newpipe.beta/) as a separate app.
+It includes not only new features such as local playlist and SoundCloud support but also an interface to detect memory leaks.
```

Comment:
Instead of "but ...", how about adding a stop, and continuing like "Furthermore, it comes with an interface ..."?